base/local/thanos-store.yaml

```diff
@@ -54,19 +55,19 @@ spec:
             path: /metrics
             port: http
             scheme: HTTP
-          periodSeconds: 30
+          periodSeconds: 20
           successThreshold: 1
-          failureThreshold: 5
+          failureThreshold: 100
```

Comment:
:astonished:

Reply:
why not?

Reply:
just amazed by how much you had to loose this in order to let start

Reply:
I just didnt want to play around with the value, so  figured why not make it big

Reply:
wouldn't that give a period of 2000 seconds where liveness probe will fail but kube won't care about it? so a good half hour.